Polyhedron/demo/Polyhedron/Scene_polygon_soup_item.h

```diff
@@ -194,7 +205,7 @@ public Q_SLOTS:
     using CGAL::Three::Scene_item::initialize_buffers;
     void initialize_buffers(CGAL::Three::Viewer_interface *viewer) const;
     void compute_normals_and_vertices(void) const;
-    void triangulate_polygon(Polygons_iterator ) const;
+    void triangulate_polygon(Polygons_iterator , int nb) const;
```

Comment:
`nb` seems to be the ID of the polygon. If that is true, then the name of that parameter is really badly chosen! It should it something like `polygon_id` or `polygonID`.